.gitignore

```diff
@@ -2,4 +2,5 @@ phpunit.xml
 composer.lock
 build
 vendor
-coverage.clover
\ No newline at end of file
+coverage.clover
+ 
```

Comment:
Please don't fix code that isn't broken.

The tool is simply wrong here, as it doesn't know how to correctly deal with `.gitignore`.

This is just a false positive.